lib/blockscore/errors/api_error.rb

```diff
@@ -15,19 +17,29 @@ class APIError < Error
     # APIError - Indicates an error on the server side (HTTP 5xx)
     # AuthenticationError - Indicates an authentication error (HTTP 401)
     def initialize(response)
-      body = JSON.parse(response.body, symbolize_names: true)
+      @http_body = JSON.parse(response.body, symbolize_names: true)
 
-      @message = body[:error][:message]
+      @message = error_advice[:message]
       @http_status = response.code
-      @error_type = body[:error][:type]
-      @http_body = body
+      @error_type = error_advice[:type]
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/56/files#r52110298